apis/facebook/api/src/facebook_api.e

```diff
@@ -124,6 +124,50 @@ feature -- Facebook: Get User
 			end
 		end
 
+	user_likes (a_path: STRING; a_params: detachable FB_PAGE_PARAMETER): detachable STRING
+		do
+			if
+				attached a_params and then
```

Comment:
just a matter of personal taste
I use "attached var" only when I have "...    as ..." 
But your code is good as well.
This is just that I prefer  `a_params /= Void` ... because else we will have  `not attached a_params` .. and I find it ugly compared to `a_params = Void`.

Reply:
Ok I will follow your style

Reply:
done